shared/configHelper.js

```diff
@@ -33,141 +33,114 @@ var path = require('path'),
+            logInfo('\n OR \n', COLOR.cyan);
+        }
+        var commandName = cli.commands[i];
+        var command = getCommandExpanded(cli, commandName);
+        logInfo('# ' + command.description, COLOR.magenta);
+        logInfo(cliName + ' ' + commandName, COLOR.magenta);
+        printArgs(cli, commandName);
     }
-    logInfo('    --templaterepouri=<Template repo URI> (e.g. https://github.com/forcedotcom/SmartSyncExplorerReactNative)]', COLOR.magenta);
-    logInfo('    --appname=<Application Name>', COLOR.magenta);
-    logInfo('    --packagename=<App Package Identifier> (e.g. com.mycompany.myapp)', COLOR.magenta);
-    logInfo('    --organization=<Organization Name> (Your company\'s/organization\'s name)', COLOR.magenta);
-    logInfo('    [--outputdir=<Output directory> (Leave empty for current directory)]', COLOR.magenta);
-    logInfo('\n OR \n', COLOR.cyan);
-    logInfo(forcecliName + ' version', COLOR.magenta);
     logInfo('\n OR \n', COLOR.cyan);
-    logInfo(forcecliName, COLOR.magenta);
+    logInfo(cliName, COLOR.magenta);
     logInfo('\nWe also offer:', COLOR.cyan);
-    for (var cli of Object.values(SDK.forceclis)) {
-        if (cli.name != forcecli.name) {
-            logInfo('- ' + cli.name + ': ' + cli.description, COLOR.cyan);
+    for (var otherCliName in SDK.forceclis) {
+        var otherCli = SDK.forceclis[otherCliName];
+        if (otherCli.name != cli.name) {
+            logInfo('- ' + otherCli.name + ': ' + otherCli.description, COLOR.cyan);
         }
     }
     logInfo('\n');
 }
 
 //
-// Processor list for 'create' command
+// Processor list
 //
-function createArgsProcessorList(forcecli, isCreateWithTemplate) {
-    var appTypes = forcecli.appTypes;
-    var platforms = forcecli.platforms;
+function createArgsProcessorList(cli, commandName) {
     var argProcessorList = new commandLineUtils.ArgProcessorList();
 
-    
-    if (platforms.length > 1) {
-        // Platforms
-        addProcessorFor(argProcessorList, 'platform', 'Enter the target platform(s) separated by commas (' + platforms.join(', ') + '):',
-                        'Platform(s) must be in ' + platforms.join(', ') + '.', 
-                        function(val) { return !val.split(",").some(p=>platforms.indexOf(p) == -1); });
-    }
-
-    if (isCreateWithTemplate) {
-        // Template Repo URI
-        addProcessorFor(argProcessorList, 'templaterepouri', 'Enter URI of repo containing template application:',
-                     'Invalid value for template repo uri: \'$val\'.', /^\S+$/);
-    }
-    else {
-        if (appTypes.length > 1) {
-            // App type
-            addProcessorFor(argProcessorList, 'apptype', 'Enter your application type (' + appTypes.join(', ') + '):',
-                            'App type must be ' + appTypes.join(', ') + '.', 
-                            function(val) { return appTypes.indexOf(val) >= 0; });
-        }
+    for (var arg of getArgsExpanded(cli, commandName)) {
```

Comment:
Same here.
All the possible params etc are in constants.js. So the code left here is simply looping over them.